README.md

```diff
@@ -0,0 +1,67 @@
+## CALCULATOR
+> A calculator project built to understand  Javascript.
+
+![Screenshot loading...](assets\screenshot.png)
```

Comment:
Your project screenshot seems not to be showing on github. Kindly find a way to ensure the the screenshot is rendering in your project Readme on GitHub
![Screenshot from 2021-10-10 18 27 24](https://user-images.githubusercontent.com/26856491/136706730-39a6e002-c018-41b8-ba46-af904fd953f4.png)